File: URI/Beginner/3302.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int t;
    cin >> t;
    int i = 1;
    while(t--)
    {
        int n;
        cin >> n;
        cout << "resposta " << i << ": " << n << "\n";
        i++;
    }
 
    return 0;
}

File: URI/Beginner/3302.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t;
        t = input.nextInt();
        int i = 1;
        while(t > 0)
        {
            int n = input.nextInt();
            System.out.println("resposta " + i + ": " + n);
            i++;
            t--;
        }
 
    }
 
}
